### PR TITLE
Feature/#1 remove member

### PR DESCRIPTION
**Pull request by BerendWouters**
Adds the long awaited feature to remove members.
Includes a confirmation dialog.

Not included, because to be discussed: audit trailing of the removal. An attempt was made by introducing a SystemAuditEvent, but it feels like I do need to validate this data model design with someone else.

### CODE REVIEW

**Comment by BerendWouters** [comment]
*bump* @vyruz1986 @Joren-Thijs

**Review by vyruz1986 — CHANGES_REQUESTED** [review]
Not a fan of the commented SystemAudit code, either make it work or put it into a separate branch for future use

**Review by vyruz1986 — CHANGES_REQUESTED** [review]
There is still a lot of SystemAuditEvent code in comments, please remove if not being used

File: Persistence/HaSpManContext.cs [hunk 1]
```diff
@@ -24,4 +28,13 @@ protected override void OnModelCreating(ModelBuilder builder)
         );
         base.OnModelCreating(builder);
     }
+    
+    //public class HaspmanContextFactory : IDesignTimeDbContextFactory<HaSpManContext>
```
Comment (vyruz1986) [inline Persistence/HaSpManContext.cs:32]: GetRidOfItRichardWilliamsGIF

**Review by vyruz1986 — CHANGES_REQUESTED** [review]

**Review by vyruz1986 — APPROVED** [review]

**Pull request merged**